apisix/core/id.lua

```diff
@@ -20,36 +20,38 @@
 -- @module core.id
 
 local fetch_local_conf = require("apisix.core.config_local").local_conf
-local try_read_attr    = require("apisix.core.table").try_read_attr
-local log              = require("apisix.core.log")
-local uuid             = require('resty.jit-uuid')
-local smatch           = string.match
-local open             = io.open
-
-
+local try_read_attr = require("apisix.core.table").try_read_attr
+local profile = require("apisix.core.profile")
+local log = require("apisix.core.log")
+local uuid = require("resty.jit-uuid")
+local smatch = string.match
+local open = io.open
+local lyaml = require("lyaml")
```

Comment:
Please put this line after `local uuid = require("resty.jit-uuid")`

Reply:
and beautify the code placing `=`s directly above each other.